omegaconf/base.py

```diff
@@ -5,7 +5,20 @@
 from typing import Any, Dict, Iterator, Optional, Tuple, Type, Union
 
 from ._utils import ValueKind, _get_value, format_and_raise, get_value_kind
-from .errors import ConfigKeyError, MissingMandatoryValue, UnsupportedInterpolationType
+from .errors import (
+    ConfigKeyError,
+    InterpolationParseError,
+    MissingMandatoryValue,
+    UnsupportedInterpolationType,
+)
+
+
+@dataclass
+class InterpolationRange:
+    """Used to store start/stop indices of interpolations being evaluated"""
+
+    start: int = -1
+    stop: int = -1
```

Comment:
could it make more sense to use Optional[int] instead of -1 here?

Reply:
Done in b3bedcb (actually making `start` a mandatory `int` to simplify type checks)

Reply:
Done in b3bedcb (actually making start a mandatory int to simplify type checks)